webpack.config.renderer.dev.js

```diff
@@ -50,6 +50,23 @@ export default merge.smart(baseConfig, {
   module: {
     rules: [
       {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: true,
+            plugins: [
```

Comment:
Should not be right, `stage-0` included `class-properties` and main process (node 7.x) doesn't need `es2015-classes`, it just a fix for ensure it put before of `react-hot-loader/babel`.